lib/kasket/configuration_mixin.rb

```diff
@@ -61,16 +63,18 @@ def kasket_indices
       result.uniq
     end
 
-    def has_kasket_index_on?(sorted_attributes)
+    def has_kasket_index_on?(sorted_attributes) # rubocop:disable Style/PredicateName
       kasket_indices.include?(sorted_attributes)
     end
 
-    def has_kasket(options = {})
```

Comment:
wanted to change it anyway since it's bad to have anyone call it with options and then have them ignored